x-pack/plugins/synthetics/common/runtime_types/ping/ping.ts

```diff
@@ -89,10 +90,15 @@ export const TlsType = t.partial({
 
 export type Tls = t.TypeOf<typeof TlsType>;
 
+export enum Status {
+  up = 'up',
+  down = 'down',
+}
+
 export const MonitorType = t.intersection([
   t.type({
     id: t.string,
-    status: t.string,
+    status: tEnum<Status>('Status', Status),
```

Comment:
Kind of a nit/question; is this necessary? Why not just declare the `t` type inline. There are no external references to `Status` here.

Reply:
i think t has no inline type, to use enum we have to use this util. At least i wasn't able to find a way.